README.md

```diff
@@ -5,14 +5,19 @@ The extension unfolds a frustum (=truncated cone) or a cone (if cut diameter=0)
 
 ## Installation
 
-Copy the two files sheet_metal_conus.inx and sheet_metal_conus.py into this folders:
+Copy the two files sheet_metal_conus.inx and sheet_metal_conus.py into the extensions folder:
+The exact folder is indicated in:
+ - Edit -> Preferences -> System: User extensions
```

Comment:
As I understand it this is now best instruction because every user will find exact correct folder on their system.